setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = gdc-rnaseq-tools
+description = RNASeq tools
+url = https://github.com/NCI-GDC/gdc-rnaseq-tool
+author = Charles Czysz
+author_email = czysz@uchicago.edu
+python_requires = >=3.6
+license_files = LICENSE
+long_description = file:README.md
+long_description_content_type = text/markdown
+classifiers =
+        License :: OSI Approved :: Apache Software License
+
+[options]
+install_requires =
+        pandas
+        numpy
+
+zip_safe = False
+include_package_data = True
+packages = find:
+
+[options.entry_points]
+console_scripts =
+	gdc_rnaseq_tools = gdc_rnaseq_tools.__main__:main
```

Comment:
Projects should now specify the `__main__.py:main()` function as the entrypoint.
The left-hand-side specifies the name of the program in the shell.